### DISCUSSION TITLE
一个关于贴图上传的小问题

### DISCUSSION THREAD

**Discussion by penndu** [ArtalkJS/Artalk#132] [Q&A] [answered]
我是用第三方图床，数据要先传到服务器上，再通过API传到图床，因为服务器的下行带宽只有一兆，所以访客反馈上传极慢。

不知道有什么方法可以解决这个问题？

**Comment by inkss** [comment]
@qwqcode 像 waline 一样，允许自定义图片上传方法吧。

https://github.com/walinejs/waline/blob/b9f491f6ec8e917fc4a2f544813d831064fb0322/packages/client/src/compact/v1.ts#L40-L52

**Reply by qwqcode** [reply, marked as answer]
https://github.com/ArtalkJS/Artalk/commit/5d8d05a291b14e4dc959bce71844b8555f41782f
